### PR TITLE
Fix local MySQL setup

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Makes it work reliably.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -140,7 +140,7 @@ tasks:
     cmds:
       # those two should always be in sync
       - go generate -x ./build/version
-      - go build -v -o bin/ ./cmd/envtool/
+      - go build -v -o bin/ {{.RACE_FLAG}} -tags={{.BUILD_TAGS}} ./cmd/envtool/
```
Comment (AlekSi) [inline Taskfile.yml:143]: Mainly for the `ferretdb_debug` build tag to capture resource stack traces

File: cmd/envtool/envtool.go [hunk 2]
```diff
@@ -170,9 +170,10 @@ func setupMySQL(ctx context.Context, logger *zap.SugaredLogger) error {
 	for ctx.Err() == nil {
 		db, err := p.Get("root", "password")
 		if err == nil {
-			if _, rowErr := db.QueryContext(ctx, "GRANT ALL PRIVILEGES ON *.* TO 'username'@'%';"); rowErr != nil {
+			if _, err = db.ExecContext(ctx, "GRANT ALL PRIVILEGES ON *.* TO 'username'@'%';"); err != nil {
```
Comment (AlekSi) [inline cmd/envtool/envtool.go:173]: `db.QueryContext`'s result rows were not finalized

**Comment by AlekSi** [comment]
@adetunjii PTAL

**Review by henvic — APPROVED** [review]

File: cmd/envtool/envtool.go (hunk 2 above)
Comment (adetunjii) [reply cmd/envtool/envtool.go:173]: Ahh yes. I see the problem now. Good catch.

**Pull request merged**